cosmos/plugin/__init__.py

```diff
@@ -219,7 +219,7 @@ def dbt_docs_index(self) -> str:
             # Hack the dbt docs to render properly in an iframe
             iframe_resizer_url = url_for(".static", filename="iframeResizer.contentWindow.min.js")
             html = html.replace("</head>", f'{iframe_script}<script src="{iframe_resizer_url}"></script></head>', 1)
-            return html
```

Comment:
If you really want a test I can try to eventually get to it, but heads up though, I'm busy all of this weekend and next week.

My 2 cents is that a test in this case seems a little low value, since this was just a hardcoded value in a return statement so it's not in the path of anything that would ever regress, plus it is a rare behavior to begin with (I'd never heard anyone mention this bug until yesterday). Hence why I didn't add a test for this behavior.

Reply:
OK, I was able to sneak in writing the unit test before leaving for work today... #1058